k9mail/src/main/java/com/fsck/k9/activity/setup/outgoing/AccountSetupOutgoing.java

```diff
@@ -82,18 +89,10 @@ public void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.account_setup_outgoing);
 
         String accountUuid = getIntent().getStringExtra(EXTRA_ACCOUNT);
-        mAccount = Preferences.getPreferences(this).getAccount(accountUuid);
-
-        try {
-            if (new URI(mAccount.getStoreUri()).getScheme().startsWith("webdav")) {
-                mAccount.setTransportUri(mAccount.getStoreUri());
-                AccountSetupCheckSettings.actionCheckSettings(this, mAccount, CheckDirection.OUTGOING);
-            }
-        } catch (URISyntaxException e) {
-            // TODO Auto-generated catch block
-            e.printStackTrace();
-        }
+        Account account = Preferences.getPreferences(this).getAccount(accountUuid);
```

Comment:
just pass the account uuid here. it's incoming data from the intent, but the view doesn't have to know what to do with it